### PR TITLE
Change semantics

### LINKED ISSUE #1495: Improve Leaderboard using semantic HTML
# Description

The leaderboard can be improved using more semantic HTML where possible.

For example the item on the leaderboard are currently divs. Instead of divs we can use a ordered list with list items.

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- For design guidelines checkout our [Figma design](https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap).
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by martinyis**
# Description
I wanted to share with you the changes I've made through a pull request to improve the accessibility and semantics of the RibbonWrapper and TreeImage components.

Firstly, in the RibbonWrapper component, I updated the Typography component to a more suitable figcaption element. This change will help convey the association between the text and image and make it easier for assistive technologies to interpret.

Secondly, in the TreeImage component, I replaced the Icon component with an img element. This change better communicates the purpose of the component, which is to display an image of a tree.

Additionally, I also converted the list of countries from a set of Paper components to an ordered list of list items, which makes the markup more semantic and easier to navigate with screen readers.

I believe these changes will greatly enhance the accessibility and SEO of the codebase. Please don't hesitate to let me know if you have any questions or concerns.
Fixes #1495

### CODE REVIEW

File: src/components/LeaderBoard.js [hunk 1]
```diff
@@ -78,25 +78,21 @@ const useStyles = makeStyles()((theme) => ({
...
 function RibbonWrapper({ fill, index }) {
   const { classes } = useStyles();
-
   return (
     <div className={classes.ribbons}>
       <Ribbon fill={fill} />
-      <Typography variant="h5" className={classes.rank}>
-        {index + 1}
-      </Typography>
+      <figcaption className={classes.rank}>{index + 1}</figcaption>
```
Comment (dadiorchen) [inline src/components/LeaderBoard.js:84]: @martinyis thanks for the contribution!
We can stay on MUI and change the tag at the same time, check this doc:
https://mui.com/material-ui/api/typography/

Comment (RubenSmn) [reply src/components/LeaderBoard.js:84]: I agree with Dadior, we can use the component prop to change the element. The same goes for the other ones.

**Review by dadiorchen — COMMENTED** [review]
@RubenSmn good, yea, let's make this a convention, use MUI prop to change the element, to avoid unnecessary HTML tag layers

**Review by dadiorchen — CHANGES_REQUESTED** [review]
@RubenSmn good, yea, let's make this a convention, use MUI prop to change the element, to avoid unnecessary HTML tag layers

**Pull request closed without merging**